mysite/search/views.py

```diff
@@ -39,7 +39,6 @@
 
 # Via http://www.djangosnippets.org/snippets/1435/
 
-
```

Comment:
PEP8 doesn't like this change: `mysite/search/views.py:42:1: E302 expected 2 blank lines, found 1`